### PR TITLE
this is a test for uploading aquila to bioconda. A formal upload will be done later.

### PR DESCRIPTION
**Pull request by LYC-vio**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [ v] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ v] This PR adds a new recipe.
* [ v] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ v] This PR does something else (explain below).
This is actually a test to see if I get everything done right before formally uploading the Aquila package.

### CODE REVIEW

**Comment by LYC-vio** [comment]
@bioconda/python 
Hi! I'm kind of new to bioconda, and I've encountered some problem in the test-linux step.
the message showed that the minimap2 package would be installed (as follows)
The following NEW packages will be INSTALLED:
21:53:05 BIOCONDA INFO (OUT) 
21:53:05 BIOCONDA INFO (OUT)     _libgcc_mutex:   0.1-main                   defaults
21:53:05 BIOCONDA INFO (OUT)     aqtest:          0.1-pyh5ca1d4c_0           local
21:53:05 BIOCONDA INFO (OUT)     bcftools:        1.9-ha228f0b_4             bioconda
21:53:05 BIOCONDA INFO (OUT)     bzip2:           1.0.8-h516909a_0           conda-forge
21:53:05 BIOCONDA INFO (OUT)     ca-certificates: 2019.6.16-hecc5488_0       conda-forge
21:53:05 BIOCONDA INFO (OUT)     certifi:         2019.6.16-py37_1           conda-forge
21:53:05 BIOCONDA INFO (OUT)     curl:            7.65.3-hf8cf82a_0          conda-forge
21:53:05 BIOCONDA INFO (OUT)     htslib:          1.9-ha228f0b_7             bioconda
21:53:05 BIOCONDA INFO (OUT)     krb5:            1.16.3-h05b26f9_1001       conda-forge
21:53:05 BIOCONDA INFO (OUT)     libblas:         3.8.0-10_openblas          conda-forge
21:53:05 BIOCONDA INFO (OUT)     libcblas:        3.8.0-10_openblas          conda-forge
21:53:05 BIOCONDA INFO (OUT)     libcurl:         7.65.3-hda55be3_0          conda-forge
21:53:05 BIOCONDA INFO (OUT)     libdeflate:      1.0-h14c3975_1             bioconda
21:53:05 BIOCONDA INFO (OUT)     libedit:         3.1.20170329-hf8c457e_1001 conda-forge
21:53:05 BIOCONDA INFO (OUT)     libffi:          3.2.1-he1b5a44_1006        conda-forge
21:53:05 BIOCONDA INFO (OUT)     libgcc-ng:       9.1.0-hdf63c60_0           defaults
21:53:05 BIOCONDA INFO (OUT)     libgfortran-ng:  7.3.0-hdf63c60_0           defaults
21:53:05 BIOCONDA INFO (OUT)     liblapack:       3.8.0-10_openblas          conda-forge
21:53:05 BIOCONDA INFO (OUT)     libopenblas:     0.3.6-h6e990d7_4           conda-forge
21:53:05 BIOCONDA INFO (OUT)     libssh2:         1.8.2-h22169c7_2           conda-forge
21:53:05 BIOCONDA INFO (OUT)     libstdcxx-ng:    9.1.0-hdf63c60_0           defaults
21:53:05 BIOCONDA INFO (OUT)     minimap2:        2.16-h84994c4_1            bioconda
21:53:05 BIOCONDA INFO (OUT)     ncurses:         6.1-hf484d3e_1002          conda-forge
21:53:05 BIOCONDA INFO (OUT)     numpy:           1.16.4-py37h95a1406_0      conda-forge
21:53:05 BIOCONDA INFO (OUT)     openblas:        0.3.6-h6e990d7_4           conda-forge
21:53:05 BIOCONDA INFO (OUT)     openssl:         1.1.1c-h516909a_0          conda-forge
21:53:05 BIOCONDA INFO (OUT)     pip:             19.2.1-py37_0              conda-forge
21:53:05 BIOCONDA INFO (OUT)     pysam:           0.15.2-py37h4b7d16d_3      bioconda
21:53:05 BIOCONDA INFO (OUT)     python:          3.7.3-h33d41f4_1           conda-forge
21:53:05 BIOCONDA INFO (OUT)     readline:        8.0-hf8c457e_0             conda-forge
21:53:05 BIOCONDA INFO (OUT)     samtools:        1.9-h8571acd_11            bioconda
21:53:05 BIOCONDA INFO (OUT)     scipy:           1.3.0-py37h921218d_0       conda-forge
21:53:05 BIOCONDA INFO (OUT)     setuptools:      41.0.1-py37_0              conda-forge
21:53:05 BIOCONDA INFO (OUT)     sqlite:          3.29.0-hcee41ef_0          conda-forge
21:53:05 BIOCONDA INFO (OUT)     tk:              8.6.9-hed695b0_1002        conda-forge
21:53:05 BIOCONDA INFO (OUT)     wheel:           0.33.4-py37_0              conda-forge
21:53:05 BIOCONDA INFO (OUT)     xz:              5.2.4-h14c3975_1001        conda-forge
21:53:05 BIOCONDA INFO (OUT)     zlib:            1.2.11-h516909a_1005       conda-forge
21:53:05 BIOCONDA INFO (OUT) 
.21:53:17 BIOCONDA INFO (OUT) Preparing transaction: ...working... done
21:53:19 BIOCONDA INFO (OUT) Verifying transaction: ...working... done
21:53:25 BIOCONDA INFO (OUT) Executing transaction: ...working... done

however, in the test step, it outputted an error

1:53:26 BIOCONDA INFO (OUT) + step1 --help
21:53:26 BIOCONDA INFO (OUT) Traceback (most recent call last):
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/bin/step1", line 6, in <module>
21:53:26 BIOCONDA INFO (OUT)     from pkg_resources import load_entry_point
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3241, in <module>
21:53:26 BIOCONDA INFO (OUT)     @_call_aside
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3225, in _call_aside
21:53:26 BIOCONDA INFO (OUT)     f(*args, **kwargs)
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3254, in _initialize_master_working_set
21:53:26 BIOCONDA INFO (OUT)     working_set = WorkingSet._build_master()
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/pkg_resources/__init__.py", line 583, in _build_master
21:53:26 BIOCONDA INFO (OUT)     ws.require(__requires__)
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/pkg_resources/__init__.py", line 900, in require
21:53:26 BIOCONDA INFO (OUT)     needed = self.resolve(parse_requirements(requirements))
21:53:26 BIOCONDA INFO (OUT)   File "/opt/conda/conda-bld/aqtest_1563918665869/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p/lib/python3.7/site-packages/pkg_resources/__init__.py", line 786, in resolve
21:53:26 BIOCONDA INFO (OUT)     raise DistributionNotFound(req, requirers)
21:53:26 BIOCONDA INFO (OUT) pkg_resources.DistributionNotFound: The 'minimap2' distribution was not found and is required by aqtest

and my meta.yaml was like this

package:
  name: aqtest
  version: "0.1"

build:
  noarch: python
  number: 0

source:
  url: https://github.com/LYC-vio/Aqtest/archive/v0.0.2.tar.gz
  sha256: 1a7ae40966dbbfc99d791f951a1f4671c0f36ff4288656a4374d19fc93cdcae9

requirements:
  build:
    - python
    - setuptools >=1.1
    - samtools
    - minimap2 ==2.16
    - scipy
    - numpy
    - pysam


  run:
    - python
    - samtools
    - minimap2 ==2.16
    - scipy
    - numpy
    - pysam

test:
  commands:
    - step1 --help
    - step2 --help
    - assemble --help
    - phasing --help

about:
  home: https://github.com/LYC-vio/Aqtest
  license: MIT
  summary: This is a simple test

Is there any way to solve this problem? 
Thanks a bunch!

**Comment by LYC-vio** [comment]
@epruesse
Hi~would you please help me review my PR: bioconda/bioconda-recipes#16508 ?
thanks a lot!

File: recipes/aqtest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  noarch: python
+  number: 0
+
+source:
+  url: https://github.com/LYC-vio/Aqtest/archive/v0.0.5.tar.gz
+  sha256: 3c269aad7d9180faaca306d1b667965e3f15a208b9de46ec4e2e0dfca5f0204b
+
+requirements:
+  build:
```
Comment (dpryan79) [inline recipes/aqtest/meta.yaml]: Please change this to `host:`

File: recipes/aqtest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - python
+    - setuptools >=1.1
+    - samtools
```
Comment (dpryan79) [inline recipes/aqtest/meta.yaml]: The only things you actually need for installation are likely python and setuptools, the rest can be relegated to `run:`.

File: recipes/aqtest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  commands:
+    - step1 --help
+    - step2 --help
+    - assemble --help
+    - phasing --help
+
+about:
+  home: https://github.com/LYC-vio/Aqtest
+  license: MIT
+  summary: This is a simple test
```
Comment (dpryan79) [inline recipes/aqtest/meta.yaml]: Please replace the test summary with the real one :)

File: recipes/aqtest/meta.yaml (hunk 2 above)
Comment (LYC-vio) [reply recipes/aqtest/meta.yaml]: thanks a bunch! :) I'm kind of new to this business, these suggestions are really helpful.

**Pull request closed without merging**